website/src/pages/extend/index.tsx

```diff
@@ -16,57 +23,63 @@ export default function Home(): JSX.Element {
         <div className="container mx-auto flex px-5 py-24 items-center justify-center flex-col">
           <div className="text-center lg:w-2/3 w-full">
             <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-4 font-medium text-gray-900 dark:text-white">
-              Extend capabilities with Docker Desktop extensions.
+              Extensibility Documentation and Resources
+            </h1>
+            <p>Explore our comprehensive guides on extending Podman Desktop:</p>
+            <div className="text-center">
+              <ul className="list-none pl-0 mt-2">
+                <li>
+                  <FontAwesomeIcon icon={faRocket} className="mr-2" />
+                  <a href={useBaseUrl('/docs/extensions/developing')}>Developing an Extension</a>
+                </li>
+                <li>
+                  <FontAwesomeIcon icon={faCloudArrowDown} className="mr-2" />
+                  <a href={useBaseUrl('/docs/extensions/install')}>Installing an Extension</a>
+                </li>
+                <li>
+                  <FontAwesomeIcon icon={faCertificate} className="mr-2" />
+                  <a href={useBaseUrl('/docs/extensions/publish')}>Publishing an Extension</a>
+                </li>
+                <li>
+                  <FontAwesomeIcon icon={faGears} className="mr-2" />
+                  <a href={useBaseUrl('/api')}>API Reference</a>
+                </li>
+              </ul>
+            </div>
+            <h1 className="mt-24 title-font sm:text-4xl text-3xl lg:text-6xl mb-4 font-medium text-gray-900 dark:text-white">
+              Extend capabilities with Podman Desktop plug-ins.
```

Comment:
```suggestion
              Extend capabilities with Podman Desktop extensions.
```

What wording should we use to be consistent ? Inside podman desktop we are using the `extension`, might be better to stick with that ?

Reply:
You are right, we should use extensions.